src/PowerBIEmbed.tsx

```diff
@@ -163,14 +163,14 @@ export class PowerBIEmbed extends React.Component<EmbedProps> {
 
 				// Set filters on the embedded report if available and different from the previous filter
 				if (filters && !isEqual(filters, prevEmbedConfig.filters)) {
-					// Upcast to Report and call setFilters
-					await (this.embed as Report).setFilters(filters);
+					// Upcast to Report and call updateFilters of replace filter type

```

Comment:
Please update this comment to 'Upcast to Report and call updateFilters with the Replace filter operation'.

And similarly `RemoveAll` for the remove operation.